src/pages/examples/3d-visual-analytics.js

```diff
@@ -0,0 +1,199 @@
+          <h2>{title}</h2>
+          <div className={classnames('intro')}>
+            <div>{description}</div>
+          </div>
+          <Link to={url} className={classnames('target')}></Link>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function BusinessInt() {
+  const context = useDocusaurusContext();
+  const { siteConfig = {} } = context;
+  return (
+    <Layout
+      title={`3D Visual Analytics Application | Vertex Developer Portal`}
+      description="Example showing how easy it is to connect external data sources to your 3D digital twin for visual analytics."
+    >
+      <header className={classnames(styles.LPhero, styles.lightHero)}>
+        <div className={classnames('container')}>
+          <div className={classnames('row')}>
+            <div className={classnames('col')}>
+              <h1>3D Visual Analytics</h1>
+              <p>
+                Connect external data sources to your 3D digital twin and 
+                deliver powerful insights with ease. 
+              </p>
+            </div>
+            <div className={classnames('col', 'col--8', 'col--offset-2')}>
+              <img src="/img/examples/visual-analytics.png" alt="" />
+            </div>
+            <div
+              className={classnames(
+                'col',
+                'col--8',
+                'col--offset-2',
+                styles.textLeft
+              )}
+            >
+              <h2 className={classnames(styles.betaCalloutHeading)}>
+                What you can do with this application
+              </h2>
+              <p className={classnames(styles.normalParagraph)}>
+                Load a Vertex sample scene and download the corresponding CSV 
+                files of test data. Simply drag and drop the CSV files onto the 
+                app, and watch as your view updates to reflect the values 
+                found in the test data. 
+              </p>
+              <div className={classnames(styles.textCenter)}>
+                <Link to={'https://visual-analytics.vertexvis.io/?clientId=08F675C4AACE8C0214362DB5EFD4FACAFA556D463ECA00877CB225157EF58BFA&streamKey=RA00uJtbA41VR9NmZvHnXSW4H8viv9AI5Vbx'} className={classnames('button button--primary')}>
+                  Launch App
+                </Link>{' '}
+                &nbsp;
+                <Link
+                  to={'https://github.com/Vertexvis/visual-analytics-demo'}
+                  className={classnames(
+                    'button button--primary button--outline'
+                  )}
+                >
+                  View on Github
+                </Link>
+              </div>
+               <h2 className={classnames(styles.betaCalloutHeading)}>
+                Instructions
+              </h2>
+              <p className={classnames(styles.normalParagraph)}>
+                Several data sets are available for you to try different 
+                analytics. The CSV files are available on Github or you can 
```

Comment:
looks like we might want that `&nbsp;` between 'you can' and the link similar to above.

![Screen Shot 2021-05-07 at 10 42 34 AM](https://user-images.githubusercontent.com/54546064/117474873-05bd3680-af21-11eb-8c22-fad1aeefa3dc.png)

Reply:
Good catch!